spec/ruby/core/string/dedup_spec.rb

```diff
@@ -0,0 +1,49 @@
+require_relative '../../spec_helper'
```

Comment:
Could you "share" the spec instead of copying it? That way they stay in sync.
That's documented at https://github.com/ruby/ruby/blob/master/spec/ruby/CONTRIBUTING.md#shared-specs and there are plenty of examples in other spec files.

Reply:
Done.